vendor/README.md

```diff
@@ -35,7 +35,8 @@ For example, when we upgrade the version of gRPC to 1.36.0 and the version of th
 we could run the linkage tool as following:
 
 ```
-$ ./gradlew :vendor:grpc-1_36_0:shadowJar
```

Comment:
Because `shadowJar` does not run `validateVendoring` task, updating this line to execute `check` which runs `validateVendoring` task.

Reply:
(and `publish` runs `check` I guess)